cli/dcoscli/config/main.py

```diff
@@ -86,28 +85,18 @@ def _set(name, value):
     """
 
     if name == "core.dcos_url":
-        return _cluster_setup(value)
+        notice = (
+            "This config property is being deprecated. "
+            "To setup the CLI to talk to your cluster, please run "
+            "`dcos cluster setup <dcos_url>`.")
+        emitter.publish(DefaultError(notice))
```

Comment:
This might be enough, but I'm not 100% sure. Where is `config` pointing if you don't have a cluster set up yet? If you do have a cluster set up (using the `cluster setup` command), will this just change the URL for whichever cluster is the active cluster? 

To verify this works, I'd want to see some extra integration tests that tested the interaction of using the deprecated commands in conjunction with the new ones.

Reply:
i think this error message is a little bit confusing
for example:
```
dcos config set core.dcos_url http://52.37.221.58
This config property is being deprecated. To setup the CLI to talk to your cluster, please run `dcos cluster setup <dcos_url>`.
[core.dcos_url]: changed from 'https://34.209.195.182' to 'http://52.37.221.58'
[core.dcos_acs_token]: removed
```
It says that this option is being deprecated and suggests to use a new way of setting up a cluster, but it also configures the CLI to talk to cluster. I think we need to rephrase the message. We probably just want to announce that this subcommand is being deprecated and in future user should be using `dcos cluster setup`

Reply:
I can rephrase it, currently it also implies that the config is deprecated, but it's actually only the way to set it which is different, how about something like this : 

> Setting the `core.dcos_url` config directly is deprecated, please use the `dcos cluster config <dcos_url>` command instead.